jfx_bridge/bridge.py

```diff
@@ -361,7 +361,13 @@ def __init__(self, local_obj):
         self.attrs = dir(local_obj)
 
     def to_dict(self):
-        return {HANDLE: self.handle, TYPE: type(self.local_obj).__name__, ATTRS: self.attrs, REPR: repr(self.local_obj)}
+        try:
+            # For objects this has not just the class, but the full module path
+            # e.g. "ghidra.program.database.ProgramDB" for currentProgram instead of just "ProgramDB"
+            _type = type(self.local_obj).typeName
```

Comment:
As mentioned, only true for Jython. I think it might make more sense for your usecase to explicitly query for the typeName object in your code, instead of embedding it here.